cime_config/usermods_dirs/NEON/defaults/user_nl_clm

```diff
@@ -19,7 +19,7 @@
 !----------------------------------------------------------------------------------
 
 flanduse_timeseries = ' '   ! This isn't needed for a non transient case, but will be once we start using transient compsets
-fsurdat = "$DIN_LOC_ROOT/lnd/clm2/surfdata_map/NEON/surfdata_hist_78pfts_CMIP6_simyr2000_${NEONSITE}_c210720.nc"
+fsurdat = "$DIN_LOC_ROOT/lnd/clm2/surfdata_map/NEON/surfdata_hist_78pfts_CMIP6_simyr2000_${NEONSITE}_c210805.nc"
 model_year_align_urbantv = 2018
 stream_year_first_urbantv = 2018
 stream_year_last_urbantv = 2019
```

Comment:
Don't we want ndep and popdens to update to the last year of the simulation (or at least the spinup)?  this would be 2020 for spinup and 2021 for transient cases.  How do we keep updating this moving forward?

Reply:
@wwieder these don't get autoupdated by any means. Right now the only way to change it is to keep changing the last year in this file (or in each case run). The last year update could also be handedl in run_neon.py though. I don't know that's any better though.

Reply:
I think this is fine for now, but as we move to making more of the real-time hindcasts they should be updated for new years of data?  Updating urban likely ins't too important and for short runs the deposition files shouldn't changethat much?